src/resources/views/base/inc/topbar_right_content.blade.php

```diff
@@ -1,6 +1,6 @@
 {{-- This file is used to store topbar (right) items --}}
```

Comment:
I just noticed something - I can't logout 😅 Under the previous theme we had a dropdown if you clicked the user profile image, to go to "Logout" and "Change user info", after the upgrade to Bootstrap v5, that stopped working. I click the top-right profile image, nothing happens:

![CleanShot 2023-02-27 at 08 50 14](https://user-images.githubusercontent.com/1032474/221494011-fc5d7a47-3eeb-4be9-adb9-07b7554ff18b.png)

Reply:
LOL - it's fixed now... I searched for all `data-toggle` attributes and replaced them with `data-bs-toggle` since that's the new way in Bootstrap 5.